### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by Hazboun6**
This is a take 2 for  PR#139 where I've ignored the PEP8 whitespace rule E225. 
These changes clean up the code for linting, but do not go as far as PR #137 . I used `autopep8` to edit whitespace but that was mostly indenting. It did not change equation spacing for instance (for real this time!). I also used `autoflake` to remove unused imports and `isort` to sort imports.

I'll add version change numbers to this PR after it is reviewed.

### CODE REVIEW

File: enterprise_extensions/hypermodel.py [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
+
 import os
 
-from enterprise import constants as const
-from enterprise.signals import signal_base
+import numpy as np
+import scipy.linalg as sl
 
 try:
-    import cPickle as pickle
```
Comment (paulthebaker) [inline enterprise_extensions/hypermodel.py:14]: It seems pickle is not used at all so the `try`/`except` block can be removed entirely.

File: enterprise_extensions/model_utils.py [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+
 import time
-import numpy as np
-import scipy.stats as scistats
 
 import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as scistats
 
 try:
```
Comment (paulthebaker) [inline enterprise_extensions/model_utils.py:9]: same extraneous `try`/`except` here.

File: enterprise_extensions/gp_kernels.py [hunk 3]
```diff
@@ -31,6 +28,8 @@ def linear_interp_basis_dm(toas, freqs, dt=30*86400):
     return U * Dm[:, None], avetoas
 
 # linear interpolation basis in time with nu^-4 scaling
```
Comment (paulthebaker) [inline enterprise_extensions/gp_kernels.py:30]: These comments should be docstrings.  That will eliminate the need for extra whitespace

File: enterprise_extensions/empirical_distr.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-from __future__ import (absolute_import, division,
-                        print_function)
+
 import logging
+
 import numpy as np
 
 try:
```
Comment (paulthebaker) [inline enterprise_extensions/empirical_distr.py:7]: after axing `Py2.x` we don't need the `cPickle` v. `pickle` checks

**Review by paulthebaker — CHANGES_REQUESTED** [review]
There are some stray try/excepts related to pickle in python 2.  There's also some weird whitespace additions due to comments that should be docstrings.

File: enterprise_extensions/model_utils.py (hunk 2 above)
Comment (paulthebaker) [reply enterprise_extensions/model_utils.py:9]: for some reason this is highlighting the wrong lines

File: enterprise_extensions/empirical_distr.py (hunk 4 above)
Comment (paulthebaker) [reply enterprise_extensions/empirical_distr.py:7]: This thinks I'm talking about the added whitespace, but its the unaltered try/except at line 7.

Comment (Hazboun6) [reply enterprise_extensions/empirical_distr.py:7]: Good call!

File: enterprise_extensions/gp_kernels.py (hunk 3 above)
Comment (Hazboun6) [reply enterprise_extensions/gp_kernels.py:30]: Oh yeah. There were a lot of them in this file. I've put them all into the docstrings.

File: enterprise_extensions/hypermodel.py (hunk 1 above)
Comment (Hazboun6) [reply enterprise_extensions/hypermodel.py:14]: Yeah, this is a weird output from `autoflake`. It knows the imports aren't needed, but isn't sure what to do about the `try/except`. I removed them in the old PR by hand, and just missed them this time.

File: enterprise_extensions/model_utils.py (hunk 2 above)
Comment (Hazboun6) [reply enterprise_extensions/model_utils.py:9]: Got it!

**Pull request merged**